src/KDSoapServer/KDSoapServerObjectInterface.h

```diff
@@ -125,6 +127,17 @@ class KDSOAPSERVER_EXPORT KDSoapServerObjectInterface
      */
     virtual void processRequestWithPath(const KDSoapMessage &request, KDSoapMessage &response, const QByteArray &soapAction, const QString &path);
 
+    typedef QPair<QString/*name*/, QString/*value*/> HttpResponseHeaderItem;
+    typedef QList<HttpResponseHeaderItem> HttpResponseHeaderItems;
+
+    /**
+     * Returns additional HTTP response header items to be added to each HTTP response header
+     * The default implementation in this base class returns an empty list
+     * Subclasses can override this method as needed
+     *
+     */
+    virtual const HttpResponseHeaderItems additionalHttpResponseHeaderItems();
```

Comment:
Please remove the "const" which doesn't bring any benefit in this particular way of being used (it even prevents move semantics from applying).

Reply:
Well, it's a habit of the old style (c++98), thanks.
And what do you think about adding "const" at the end of the function? It seems to be better